.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         run: sbt ++${{ matrix.scala }} scalastyle "test:scalastyle"
   publish:
     needs: build
-    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+    if: contains(github.ref, 'refs/tags/v')
```

Comment:
Removing the `refs/heads/master` would disable the snapshot builds (which I find rather nice). Could you change this to trigger for both?

Reply:
Done.